### ISSUE TITLE
strip referenced by name

### ISSUE THREAD

**Issue by diramazioni** [Starfall-Robles/Blender-2-G-MIC#1]
Hi, thanks for this alpha version! I found some problems on the current implementation that maybe you are already aware, but I'll open a separate issue for each to keep track.

It seems that ImageSequence.name is used instead of SequenceElement.filename to refer to the file to process in gmic. 
In other word if I change the name of the imported image sequence by cutting the strip or renaming, it doesn't work anymore.

**Comment by Starfall-Robles** [comment]
This is correct. There are some issues reliably accessing the Strip Input filename versus the Edit Strip name...so it's strip.name for now.

This version of B2G is primarily intended to provide Blender users access to some of the powerful (and fun) features of G'MIC until the more sophisticated "bridge" plugin appears for Natron, Fusion, etc... (see https://plus.google.com/+TobiasFleischer). That bridge code should allow you to access virtually all of the G'MIC filters, and hopefully provide the editing/keyframing options you are looking for. Thanks for the feedback.
